### PR TITLE
(WIP) Redesign

### LINKED ISSUE #24: Search filters
There should be an option to filter searches to one agency/OIG.

### LINKED ISSUE #68: What should the homepage have on it?
Though I'm expecting most of our traffic to be from notifications and landing pages, the home page is still pretty important. It's just thrown together right now, and could use some clearer direction.

### PR DESCRIPTION
**Pull request by divergentdave**
Fixes #24 
Fixes #68

### CODE REVIEW

**Comment by divergentdave** [comment]
The homepage has reached reasonable facsimile status! /cc @heyitsolivia 
homepage

**Comment by divergentdave** [comment]
FWIW, I have checked the Chosen distributable files into the repository, to avoid dealing with Bower. I'm going to keep an eye on harvesthq/chosen#2550, in which they're working on publishing to an NPM package.

File: app.js [hunk 1]
```diff
@@ -13,6 +14,21 @@ if (config && config.http && config.http.port)
...
+  fromStream.pipe(toStream);
+}
+copy('node_modules/nodep-date-input-polyfill/nodep-date-input-polyfill.dist.js',
+     'public/scripts/nodep-date-input-polyfill.dist.js');
+copy('node_modules/jquery/dist/jquery.min.js',
+     'public/scripts/jquery.min.js');
+copy('node_modules/jquery/dist/jquery.min.map',
+     'public/scripts/jquery.min.map');
+copy('node_modules/jquery/dist/jquery.js',
+     'public/scripts/jquery.js');
```
Comment (konklone) [inline app.js]: Not asking you to do this before merge (this applies to any comments I leave here), but I'd prefer to just hardcode these files into the repo than run system disk access calls on app startup.

Comment (divergentdave) [inline app.js]: Easy enough! Done in 32ff8af

**Comment by konklone** [comment]
I made a few commits just to fix some spacing and copy nits. This is outstanding and moves us way forward, thank you!

**Comment by divergentdave** [comment]
:+1: thanks, good changes!

**Pull request merged**